pinot-common/src/test/java/org/apache/pinot/common/data/DateTimeFormatSpecTest.java

```diff
@@ -271,4 +271,58 @@ public Object[][] provideTestConstructFormatData() {
     });
     return entries.toArray(new Object[entries.size()][]);
   }
+
+
+  @Test(dataProvider = "testConstructFormatDataProviderPipe")
```

Comment:
Also add tests if data is getting parsed correctly such as `testFromFormatToMillis`

Reply:
Have added remaining tests.